apps/opencs/model/world/collection.hpp

```diff
@@ -120,6 +120,8 @@ namespace CSMWorld
 
             virtual const Record<ESXRecordT>& getRecord (int index) const;
 
+            virtual Record<ESXRecordT>& getRecord (int index);
```

Comment:
Do we really need this one? That would give us two methods to modify the record (non-const getRecord and setRecord).